src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_XPS_Q8.py

```diff
@@ -0,0 +1,294 @@
+    # TODO add your particular attributes here if any
+
+    """
+    _controller_units = 'mm'  
+    is_multiaxes = False  
+    _axis_names = ['Axis1'] 
+    _epsilon = 600e-6  
+    data_actuator_type = DataActuatorType['DataActuator']  
+
+
+    params = [{'title':'XPS IP address :', 'name':'xps_ip_address', 'type' : 'str', 'default' : '192.168.0.254'}, #IP address of my system
+              {'title':'XPS Port :', 'name':'xps_port', 'type' : 'int', 'default' : 5001}, #Port of my system, should be the same for others ?
+              {'title':'Group :', 'name':'group', 'type' : 'str', 'default' : 'Group2'},    #Group to be moved
+              {'title':'Positionner :', 'name':'positionner', 'type' : 'str', 'default' : 'Pos'}    #positionner to be moved
+                ] + comon_parameters_fun(is_multiaxes, axis_names=_axis_names, epsilon=_epsilon)
+
+    def ini_attributes(self):
+        self.controller: XPSPythonWrapper = None
+
+    def get_actuator_value(self):
+        """Get the current value from the hardware with scaling conversion.
+
+        Returns
+        -------
+        float: The position obtained after scaling conversion.
+        """
+        pos = DataActuator(data=self.controller.getPosition())  # when writing your own plugin replace this line
+        pos = self.get_position_with_scaling(pos)
+        return pos
+
+    def close(self):
+        """Terminate the communication protocol"""
+        self.controller.closeTCPIP()
+
+    def commit_settings(self, param: Parameter):
+        """Apply the consequences of a change of value in the detector settings
+
+        Parameters
+        ----------
+        param: Parameter
+            A given parameter (within detector_settings) whose value has been changed by the user
+        """
+        if param.name() == 'xps_ip_address':
+            self.controller.setIP(param.value())
+            self.controller.retryConnection()
+        elif param.name() == 'xps_port':
+            self.controller.setPort(param.value())
+            self.controller.retryConnection()
+        elif param.name() == 'group':
+            self.controller.setGroup(param.value())
+        elif param.name() == 'positionner':
+            self.controller.setPositionner(param.value())
+        else:
+            pass
+
+    def ini_stage(self, controller=None):
+        """Actuator communication initialization
+
+        Parameters
+        ----------
+        controller: (object)
+            custom object of a PyMoDAQ plugin (Slave case). None if only one actuator by controller (Master case)
+
+        Returns
+        -------
+        info: str
+        initialized: bool
+            False if initialization failed otherwise True
+        """
+
+        self.controller = self.ini_stage_init(old_controller=controller,
+                                              new_controller=XPSPythonWrapper(
+                                                  ip = self.settings.child('xps_ip_address').value(),
```

Comment:
to access only value of a parameter use the shortcut:
```
self.settings['xps_ip_address']
```

Reply:
thanks, done on commit [6390e14](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/6390e14ec43556b665ca15bfbb67280992660fe8)